src/main/java/com/cburch/logisim/std/ttl/Ttl74266.java

```diff
@@ -52,10 +49,10 @@ public void paintInternal(InstancePainter painter, int x, int y, int height, boo
   @Override
   public void propagateTtl(InstanceState state) {
     for (byte i = 2; i < 6; i += 3) {
-      state.setPort(i, (state.getPortValue(i - 1).xor(state.getPortValue(i - 2)).not()), 1);
+      state.setPort(i, (state.getPortValue(i - 1).xor(state.getPortValue(i - 2)).not() == Value.TRUE)?Value.UNKNOWN:Value.FALSE, 1);
```

Comment:
Please add spaces around operators `?` and `:`.

Reply:
done